cleverhans_tutorials/tutorial_models.py

```diff
@@ -87,13 +94,16 @@ def set_input_shape(self, input_shape):
                                                    self.output_channels)
         assert len(kernel_shape) == 4
         assert all(isinstance(e, int) for e in kernel_shape), kernel_shape
-        init = tf.random_normal(kernel_shape, dtype=tf.float32)
-        init = init / tf.sqrt(1e-7 + tf.reduce_sum(tf.square(init),
-                                                   axis=(0, 1, 2)))
+        if self.weight_init == 't_normal':
```

Comment:
Duplicated code here and in linear layer. It would be nice if you can get rid of code duplication.